### PR TITLE
chore(android_intent_plus): fixes dangling dart doc comment linting issue

### PR DESCRIPTION
**Pull request by OutdatedGuy**
## Description

Fixes the dangling dart doc comment linting issue by moving the ignore comment at top of file and adding one where not present. 

They do not pose any issue while using the package, but while fixing another issue in one of the packages I saw these linting warnings and thought of fixing them.

## Related Issues

Trivial changes so doesn't need an issue.

## Checklist

- [x] I read the [Contributor Guide](https://github.com/fluttercommunity/plus_plugins/blob/main/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I titled the PR using [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0).
- [x] I did not modify the `CHANGELOG.md` nor the plugin version in `pubspec.yaml` files.
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate that with a `!` in the title as explained in [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0)).
- [x] No, this is *not* a breaking change.

### CODE REVIEW

File: packages/device_info_plus/device_info_plus/lib/src/device_info_plus_macos.dart [hunk 1]
```diff
@@ -1 +1,2 @@
+// ignore: dangling_library_doc_comments
```
Comment (miquelbeltran) [inline packages/device_info_plus/device_info_plus/lib/src/device_info_plus_macos.dart:1]: Can you make this change on a separate PR?

That way, we have a PR per package.

**Review by miquelbeltran — CHANGES_REQUESTED** [review]
The change in `android_intent_plus` can be merged, but the change in `device_info_plus` should go in another PR if possible.

**Comment by OutdatedGuy** [comment]
@miquelbeltran I have created 2 separate PRs as one was fixing the linting warning and other is just simply ignoring the warning.

**Review by miquelbeltran — APPROVED** [review]
Thank you! merging this one

**Pull request merged**